app/lib/library/include/library.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef LIBRARY_HPP
+#define LIBRARY_HPP
+
+#include <Arduino.h> // Font for display
+
+class Library{
+
+    private:
+        String * fileArray;
+        static const int SIZE = 20;

```

Comment:
We should look into dynamically creating this array. If there are more than 20 files in the directory, this will cause the application to crash. We could look into getting the file count before initializing the array?